bin/pygrb/pycbc_pygrb_page_tables

```diff
@@ -649,6 +658,9 @@ if found_missed_file is not None:
              'rec_mass2', 'rec_mchirp', 'inclination', 'ra', 'dec', 'rec_ra',
              'rec_dec', 'coherent_snr', 'chisq', 'null_snr']
     keys += sngl_snr_keys
+    for key in keys:
+        if key not in found_after_vetoes.keys():
+            found_after_vetoes[key] = np.array([])
```

Comment:
I messed up my description, sorry: this goes after the next `keys+=...` so the loop is on all `keys`.